features/steps.py

```diff
@@ -0,0 +1,181 @@
+# -*- coding: utf-8 -*-
+from contextlib import closing
+import datetime
+import os
+from journal import INSERT_ENTRY
+from journal import connect_db
+from journal import DB_SCHEMA
+from lettuce import world
+from lettuce import step
+from lettuce import before
+from lettuce import after
+
+TEST_DSN = 'dbname=test_learning_journal user=nbeck'
+
+
+def init_db(setup):
+    with closing(connect_db(setup)) as db:
+        db.cursor().execute(DB_SCHEMA)
+        db.commit()
+
+
+def empty_db(setup):
+    with closing(connect_db(setup)) as db:
+        db.cursor().execute("DROP TABLE entries")
+        db.commit()
+
+
+def clear_entries(setup):
+    with closing(connect_db(setup)) as db:
+        db.cursor().execute("DELETE FROM entries")
+        db.commit()
```

Comment:
You should be able to simply import the above three functions from the `test_journal.py` module instead of re-writing them here.  It'd be more DRY to do so.